progenitor-impl/src/lib.rs

```diff
@@ -653,12 +653,23 @@ pub fn space_out_items(content: String) -> Result<String> {
     })
 }
 
+fn validate_openapi_spec_version(spec_version: &str) -> Result<()> {
+    // progenitor currenlty only support OAS 3.0.x
+    // (a limitation coming from using the openapiv3 crate, see
```

Comment:
This isn't quite accurate. While openapiv3 doesn't support 3.1, there's significant additional work required to support the incompatible changes made a to the schema definition. I would suggest dropping this parenthetical.